components/network/openvpn/files/openvpn.xml

```diff
@@ -1,8 +1,69 @@
 <?xml version="1.0" ?>
 <!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+
```

Comment:
Move these bits to other location, e.g. /usr/share/doc/openvpn/README.openindiana and just reference that file here.